api/app/utils/cffdrs.py

```diff
@@ -0,0 +1,122 @@
+""" This module contains functions for computing fire weather metrics.
+"""
+import logging
+import app.utils.r_importer
+from app.utils.singleton import Singleton
+
+
+logger = logging.getLogger(__name__)
+
+
+@Singleton
+class CFFDRS():
```

Comment:
As mentioned, I don't know that this has to be a singleton, but given time constrains I don't mind if it stays this way.